docs/firefighting/index.md

```diff
@@ -809,6 +811,36 @@ OK
 root@redis0:/opt/data/redis# rm temp-rewriteaof-\*.aof
 ```
 
+### Is your disk at least 3x your maxmemory?
+
+We use the [default AOF auto-rewrite configuration](https://github.com/redis/redis/blob/5.0/redis.conf#L757-L770), which is to rewrite the AOF (on-disk replica of in-memory redis data) whenever it doubles in size. Thus disk usage will sawtooth between X and 3X where X is the size of the AOL after rewrite: X right rewrite, 2X when rewrite is triggered, and 3X when the 2X-sized file has been written to a 1X-sized file, but the 2X-sized file has not yet been deleted, followed finally again by X after rewrite is finalized and the old file is deleted.
+
+Since the post-rewrite AOF will only ever contain as much data as is contained in redis memory, and the amount of data contained in redis memory has an upper bound of the maxmemory setting, you should make sure that your disk is at least 3 * maxmemory + whatever the size of the OS install is. Since disk is fairly cheap, give it a comfortable overhead for log files etc.
+
+## Checking redis after restart
+
+Redis takes some time to read the AOF back into memory upon restart/startup. To check if it's up, you can run the following:
+
+```memory
+$ cchq <env> ssh ansible@redis
+
+% redis-cli
+127.0.0.1:6379> ping
+(error) LOADING Redis is loading the dataset in memory
+127.0.0.1:6379> ping
+(error) LOADING Redis is loading the dataset in memory
+127.0.0.1:6379> ping
+PONG
+```
+once it responds with `PONG` redis is back up and ready to serve requests.
+
+## Tail the redis log
+
+To show the last few lines of the redis log during firefighting you can run:
+```
+cchq <env> run-shell-command redis 'tail /opt/data/redis/redis.log'
```

Comment:
Might want to add the cloudwatch logs too for more/past info.

Reply:
Okay yeah good point, will add

Reply:
https://github.com/dimagi/commcare-cloud/pull/4314